### ISSUE TITLE
A call for help with current code and problems in the paper

### ISSUE THREAD

**Issue by SUNSMILESSUN** [bbdamodaran/HSIC_FS#1]
Hello, I am very interested in the content of your paper and hope to use it for comparison algorithm.
However, I met some problems when reading this paper and running the code. I hope I can get your help.
1.The code for SK is missing a function called invChol_mex, could you share the source code of this function? Or maybe you can tell me what this function does.
2.The experimental part of the paper introduces the division of training set and test set. But there is no specific introduction of their classification criteria, may I ask what is his classification principle?
3.Because there are only training sets in the source code, I want to ask whether the Data used to select the feature subset is the training set of the source code or the entire original data set.
I sincerely hope to get your answer.
